venue/index.md

```diff
@@ -100,6 +139,8 @@ The local organising committee [has shared](https://umap.openstreetmap.fr/en/map
 
 Nairobi is generally considered low-risk for malaria, particularly within the city limits. However, it's always wise to take basic precautions, such as using insect repellent and wearing long-sleeved clothing in the evenings, to minimise any risk of mosquito bites. Other health considerations include ensuring that routine vaccinations are up-to-date and practising good hygiene to prevent common illnesses. Should you experience any health issues or have concerns, the city’s public healthcare facilities are well-equipped to assist you. In case of health emergencies, we recommend visiting private health facilities which also accept comprehensive travel insurance covers. If one needs to purchase prescription medication, one can visit a pharmacy that are common across neighbourhoods and in malls. One can also place an order on [myDAWA](https://mydawa.com/) for delivery services.
 
+![Emergency Contacts](img/emergency.svg)
```

Comment:
@mugeha-laura I noticed that the image file for this is currently still missing. Can you please create a dedicated PR to add this SVG tohttps://github.com/openstreetmap/stateofthemap-2024/tree/main/img?

Reply:
I have just done so a few minutes ago @tyrasd